spyder/widgets/about.py

```diff
@@ -105,70 +161,136 @@ def __init__(self, parent):
+                    <b>Spyder IDE</b>
+                    <br>{spyder_ver}
+                    <br> {revision}
+                    <br> """).format(
+                spyder_ver=versions['spyder'],
+                revision=revlink,
+                font_family=font_family,
+                font_size=font_size))
+        self.info.setAlignment(Qt.AlignHCenter)
 
-        btn = QPushButton(_("Copy to clipboard"), )
+        btn = QPushButton(_("Copy version info"), )
         bbox = QDialogButtonBox(QDialogButtonBox.Ok)
+        bbox.setStyleSheet(
+           f"font-size: {buttons_font_size};"
+           f"padding: {buttons_padding}"
+         )
+        btn.setStyleSheet(
+           f"font-size: {buttons_font_size};"
+           f"padding: {buttons_padding}"
+         )
 
         # Widget setup
         self.setWindowIcon(ima.icon('MessageBoxInformation'))
         self.setModal(False)
 
         # Layout
-        tophlayout = QHBoxLayout()
-        tophlayout.addWidget(self.label_pic)
-        tophlayout.addWidget(self.label)
+        piclayout = QVBoxLayout()
+        piclayout.addWidget(self.label_pic)
+        piclayout.addWidget(self.info)
+        piclayout.setContentsMargins(20, 0, 15, 0)
+
+        scroll_overview = QScrollArea(self)
+        scroll_overview.setWidgetResizable(True)
+        scroll_overview.setWidget(self.label_overview)
+
+        scroll_community = QScrollArea(self)
+        scroll_community.setWidgetResizable(True)
+        scroll_community.setWidget(self.label_community)
+
+        scroll_legal = QScrollArea(self)
+        scroll_legal.setWidgetResizable(True)
+        scroll_legal.setWidget(self.label_legal)
+
+        self.tabs = QTabWidget()
+        self.tabs.addTab(scroll_overview, _('Overview'))
+        self.tabs.addTab(scroll_community, _('Community'))
+        self.tabs.addTab(scroll_legal, _('Legal'))
+        self.tabs.setStyleSheet(
+            f"background-color: {dialog_background_color}")
+        tabslayout = QHBoxLayout()
+        tabslayout.addWidget(self.tabs)
+        tabslayout.setSizeConstraint(tabslayout.SetFixedSize)
+        tabslayout.setContentsMargins(0, 15, 15, 0)
 
         btmhlayout = QHBoxLayout()
         btmhlayout.addWidget(btn)
-        btmhlayout.addStretch()
         btmhlayout.addWidget(bbox)
+        btmhlayout.setContentsMargins(100, 20, 0, 20)
+        btmhlayout.addStretch()
 
-        vlayout = QVBoxLayout(self)
-        vlayout.addLayout(tophlayout)
-        vlayout.addSpacing(25)
+        vlayout = QVBoxLayout()
+        vlayout.addLayout(tabslayout)
         vlayout.addLayout(btmhlayout)
         vlayout.setSizeConstraint(vlayout.SetFixedSize)
 
+        mainlayout = QHBoxLayout(self)
+        mainlayout.addLayout(piclayout)
+        mainlayout.addLayout(vlayout)
+
         # Signals
         btn.clicked.connect(self.copy_to_clipboard)
         bbox.accepted.connect(self.accept)
+        self.resize(550, 430)
+        css = APP_STYLESHEET.get_copy()
```

Comment:
```suggestion
        # Size
        self.resize(550, 430)

        # Style
        css = APP_STYLESHEET.get_copy()
```

Small stylistic suggestion.